tests/integration/helpers.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+"""Helper functions for Discourse charm tests."""
```

Comment:
Shouldn't we keep the copyright notice before the docs?

Reply:
Good point, fixed!